src/test/resources/bundles/defaultProcess_V1/config.json

```diff
@@ -85,6 +85,7 @@
         "state": "responseState"
       },
       "templateName": "question",
+      "showDetailCardHeader" : true,
```

Comment:
The default value is true so it is not needed, it "weighs down" unnecessarily the file.